include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -230,6 +230,18 @@ class FEEvaluationBaseData
   ArrayView<VectorizedArrayType>
   get_scratch_data() const;
 
+  /**
+   * Return the number of the quadrature formula of the present cell.
+   */
+  unsigned int
+  get_quadrature_index() const;
+
+  /**
+   * Return index of cell or face.
+   */
+  unsigned int
+  get_index() const;
```

Comment:
Can we call this `get_current_cell_index()` or something similarly to indicate that it is the index we look at? I agree there is ambiguity due to the cell or face behavior defined in a derived class.

Reply:
Done.